src/game/common/gamelod.cpp

```diff
@@ -418,7 +418,10 @@ int GameLODManager::Get_Recommended_Texture_Reduction()
     }
 
     if (m_didMemPass) {
-        return m_staticLOD[m_idealStaticGameDetail].texture_reduction_factor;
+        // #BUGFIX Test to avoid invalid indexing
+        if (m_idealStaticGameDetail > STATLOD_INVALID && m_idealStaticGameDetail < STATLOD_COUNT) {
```

Comment:
shouldn't this be `>= 0`. `STATLOD_INVALID` is currently -1 but if it ever became -2 or a different value then this check would stop being a bounds check.

Reply:
Hmm good point. But its already committed.